insns.def

```diff
@@ -701,6 +701,20 @@ defined
     }
 }
 
+/* defined?(@foo) */
+DEFINE_INSN
+defined_ivar
```

Comment:
YARV instruction name has a rule "do not contain `_` other than optimization instructions".
Can I rename it?

Reply:
I would consider this an optimization instruction? But apart from that, totally fine with me renaming it.

(...though I'll defer to more experienced committers to decide here...)

Reply:
> I would consider this an optimization instruction?

with compile option optimized instructions should be disable.
anyway I'll rearrange it.
Thank you.